mozilla/fx-private-relay
# MPP-3545: Styling updates to email banners

This PR fixes MPP-3545.


# New feature description
Wrapper email styling was updated. Also added a minor change to reply banner styling to remove the margin when there is overflow.

# Screenshot (if applicable)

https://github.com/mozilla/fx-private-relay/assets/59676643/56686e53-1c97-4475-bf9a-943e36e5a6c2

# How to test
* Go to emails/wrapped_email_test
* Verify that the top banner matches the screenshots in for desktop and mobile https://mozilla-hub.atlassian.net/browse/MPP-3545

# Checklist (Definition of Done)
